addon/components/sort-pane.hbs

```diff
@@ -1,10 +1,11 @@
 <div class="sort-pane {{if @isDisabled "disabled"}}"
-    sort-pane={{if this.sortPane true}}
+    sort-pane={{@sortPaneSelector}}
```

Comment:
giving custom sort-pane selector so that if nested sortable addon is used, it doesn't cause problems